translations/base-es.yaml

```diff
@@ -590,11 +589,11 @@ storyRewards:
     no_reward:
         title: Siguiente Nivel
         desc: >-
-            Este nivel no da recompensa, pero el siguiente si! <br><br> PS: Mejor no destruyas la fábrica que tienes - ¡Necesitarás <strong>todas</strong> esas figuras más adelante para <strong>desbloquear mejoras</strong>!
+            Este nivel no da recompensa, ¡pero el siguiente si! <br><br> PS: Mejor no destruyas la fábrica que tienes - ¡Necesitarás <strong>todas</strong> esas figuras más adelante para <strong>desbloquear mejoras</strong>!
     no_reward_freeplay:
         title: Siguiente Nivel
         desc: >-
-            ¡Felicidades! ¡Por cierto, más contenido está planeado para el juego completo!
+            ¡Felicidades! ¡Por cierto, hay más contenido está planeado para el juego completo!
```

Comment:
This is grammatically wrong, the right sentence is: "¡Por cierto, hay más contenido planeado para el juego completo!"
without the "está"